examples/end-to-end-examples/sec_10k_qa/README.md

```diff
@@ -0,0 +1,190 @@
+
+The `convert_10ks_to_mds.py` script will convert the data from step 1 into the MDS format. In addition to converting to MDS, it will concatenate samples together (separated by a special "end of sequence" token) to process as many tokens as possible in each sample. We will make use of the [`ConcatTokensDataset`](https://github.com/mosaicml/llm-foundry/blob/2733446bccce1800400b3a72a718bcc1fbff3da3/llmfoundry/data/data.py#L32) in LLM-foundry, and the [`MDSWriter`](https://docs.mosaicml.com/projects/streaming/en/latest/api_reference/generated/streaming.MDSWriter.html#streaming.MDSWriter) to perform this conversion. We will use the `mosaicml/mpt-7b` tokenizer, which is the same as `EleutherAI/gpt-neox-20b`, (`--tokenizer` argument), `<|endoftext|>` to separate examples in concatenated samples (`--eos_text` argument), and a maximum sequence length of 2048 (`--concat_tokens` argument).
+
+**Fields to replace with your values:** `REPLACE_WITH_YOUR_CLUSTER`, `CLOUD`, `BUCKET_NAME` (replace `large_full` with `small_full` if you would like to run through the tutorial more quickly)
+
+**Inputs:** the `sec-10ks-large` folder from step 1
+
+**Command:** `mcli run -f mcli-yamls/02_convert_10ks_to_mds.yaml`
+
+**Outputs:** the `sec-10ks-large-mds` folder on your cloud object store, containing train, validation, and test splits of the 10k data, concatenated, tokenized, and converted to MDS format
+
+
+## 3) Finance Finetune MPT-7b
+
+Next, we will finetune our pretrained base model on the train split of the 10-K data in order to tune it on data that is in-domain for the end task of answering questions about 10-K forms. This process is called "domain tuning," and can be useful for adapting a model that has already been trained on a huge amount of data (e.g. MPT-7b) to a new domain. For this example, we will use the train/validation/test splits provided with the dataset, which splits by company. So each company (which has multiple years of 10-K forms) will only appear in one of the splits. We will use the validation split as validation data, and reserve the test split for our final testing of our application.
+
+Please check out the [training yaml](./mcli-yamls/03_finetune_on_10ks.yaml) for all of the details. This yaml will load the pretrained weights for `mpt-7b` available on the [HuggingFace Hub](https://huggingface.co/mosaicml/mpt-7b), and then train using the normal causal language modeling objective on the 10-K form dataset that we processed in the previous step. The [training script](https://github.com/mosaicml/llm-foundry/blob/main/scripts/train/train.py) itself, is from LLM-foundry.
```

Comment:
Should we add a pointer to the MCLI docs that explain the YAML schema for reference here? First time users will probably blindly run stuff but it would be good to take an opportunity to do some explanation of the contents of the YAML so they have a basic understanding of what's going on.

Also I think people find it confusing that the `parameters` section ends up as the `/mnt/config/parameters.yaml` in the pod so we may want to just call that out.

Reply:
All of this is described in the "setup" section, which hopefully people read?

Reply:
I'm not sure how much I can do if they don't read setup :P